### ISSUE TITLE
Setting qpm, defaultQpm and bpm

### ISSUE THREAD

**Issue by marina-ferreira** [paulrosen/abcjs#655]
Hi!

Thank you for this amazing library!

I found this issue concerning `qpm` #461. Thought I might be using an old version setup, but both `qpm` and `defaultQpm` are still in the documentation.

I tried setting them both on `setTune` but neither have effects.

Also, would there be a way to set the `bpm` directly, like `synthControl.setBpm(60)`, instead of setting the percentage through `setWarp`? The playback is in sync with other elements and I can't set the bpm to `60`, for example, it's either `59` or `61`.

```js
synth
  .init({
    audioContext: audioContext,
    visualObj: display[0],
    options: abcOptions.synth.options
  })
  .then(() => {
    synthControl.setTune(display[0], true, {
      audioContext: audioContext,
      onEnded: () => {},
      options: { defaultQpm: 120, qpm: 120 }
    })
    synth.prime()
  })
  .catch(error => console.log(error))

  const abcOptions = {
    render_options: {
      oneSvgPerLine: true,
      responsive: 'resize',
      add_classes: true,
      staffwidth: 500,
      paddingtop: 120,
    },
    synth: {
      el: '#audio',
      options: {
        displayLoop: true,
        displayRestart: true,
        displayPlay: true,
        displayProgress: true,
        displayWarp: true,
      }
    }
  }
```

**Comment by paulrosen** [comment]
Very curious - it looks like I broke that option when I worked on "warp". I will fix it for the next beta, hopefully soon.

In the meantime, a work around is to add the tempo in the abc string in the header with `Q:1/4=60`. You can add `%%printTempo false` if you don't want to display it.

I've had other requests for setting warps in other units. I will work on that, too.

Thanks for the bug report!

**Comment by marina-ferreira** [comment]
Thanks for the answer and tips! 😀

**Comment by paulrosen** [comment]
Will be fixed in beta 33.

**Comment by paulrosen** [comment]
Fixed in 6.0.0-beta.33
